sdk/synapse/azure-analytics-synapse-accesscontrol/src/main/java/com/azure/analytics/synapse/accesscontrol/AccessControlClient.java

```diff
@@ -0,0 +1,265 @@
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.PagedResponse;
+import com.azure.core.http.rest.PagedResponseBase;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.RestProxy;
+import com.azure.core.http.rest.SimpleResponse;
+import com.azure.core.util.Context;
+import com.azure.core.util.FluxUtil;
+import java.util.List;
+import reactor.core.publisher.Mono;
+
+/**
+ * Initializes a new instance of the synchronous AccessControlClient type.
+ */
+@ServiceClient(builder = AccessControlClientBuilder.class)
+public final class AccessControlClient {
+    private AccessControlsImpl serviceClient;
+
+    /**
+     * Initializes an instance of AccessControls client.
+     */
+    AccessControlClient(AccessControlsImpl serviceClient) {
+        this.serviceClient = serviceClient;
+    }
+
+    /**
+     * Create role assignment.
+     * 
+     * @param options Role Assignment request details.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws ErrorContractException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return role Assignment response details.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public RoleAssignmentDetails createRoleAssignment(RoleAssignmentOptions options) {
+        return this.serviceClient.createRoleAssignment(options);
+    }
+
+    /**
+     * Create role assignment.
+     * 
+     * @param options Role Assignment request details.
+     * @param context The context to associate with this operation.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws ErrorContractException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return role Assignment response details.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public RoleAssignmentDetails createRoleAssignment(RoleAssignmentOptions options, Context context) {
+        return this.serviceClient.createRoleAssignment(options, context);
+    }
+
+    /**
+     * List role assignments.
+     * 
+     * @param roleId Synapse Built-In Role Id.
+     * @param principalId Object ID of the AAD principal or security-group.
+     * @param continuationToken Continuation token.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws ErrorContractException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return a list of role assignments.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public List<RoleAssignmentDetails> getRoleAssignments(String roleId, String principalId, String continuationToken) {
```

Comment:
What is the continuation token here? How is it used?

Reply:
It indicates whether there are more results.